.travis.yml

```diff
@@ -12,25 +10,9 @@ gemfile:
   - Gemfile
   - gemfiles/Gemfile.rails-5.2-rc1
```

Comment:
WDYT about renaming this file to `gemfiles/Gemfile.rails-5.2-stable` since it's stable now?